lib/honeybadger/plugins/rails.rb

```diff
@@ -68,6 +69,19 @@ def self.source_ignored?(source)
           end
         end
       end
+
+      Plugin.register :rails_metrics do
```

Comment:
What about changing this plugin's name to just `rails`? Using `rails_metrics` feels a little awkward to me — I know we are reporting metrics, but we are reporting them as events, not as collected metrics, as we do elsewhere in this PR when we use the term metrics.

```suggestion
      Plugin.register :rails do
```